source/query/queries.txt

```diff
@@ -6,6 +6,14 @@ View Past Queries
 
 .. default-domain:: mongodb
 
+Starting in |compass| 1.31, you can view your favorite and saved 
+aggregation queries on the "My Queries" view once connected to 
+your cluster.
+
+.. figure:: /images/compass/query-my-queries-splash.png
+   :figwidth: 720 px
+   :alt: My queries view
+
```

Comment:
I'm slightly confused about the placement of this content. The title of the page is "View Past Queries", but it seems to discuss favorite queries. We have a sub-page under this one for [favorite queries](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-21583/query/favorites/). I feel like we should also be updating that page as part of these changes?

Reply:
I spoke with Rhys previously to get a better idea of the ask for this ticket. This is the page he wanted the update on since we are trying to emphasize the new screen / experience of not having to go to the favorite/history view.

Reply:
So then maybe this page title should be updated to something like "View Favorite Queries" or something more general? The title doesn't really seem to match the content.